examples/angular_bazel_architect/package.json

```diff
@@ -8,7 +8,8 @@
     "test": "bazel test :test",
     "lint": "bazel run :lint",
     "e2e": "bazel test :e2e",
-    "postinstall": "patch-package && ngcc --properties es2015 browser module main --first-only"
+    "postinstall": "patch-package && ngcc --properties es2015 browser module main --first-only",
+    "webdriver-update": "webdriver-manager update --standalone false --gecko false --versions.chrome 85.0.4183.38"
```

Comment:
Shouldn't you call `webdriver-update` post-install?

Reply:
thx, added `&& yarn webdriver-update` after patch-package